pkgs/servers/jackett/default.nix

```
@@ -1,32 +1,43 @@
 { lib
+, stdenv
 , buildDotnetModule
 , fetchFromGitHub
 , dotnetCorePackages
 , openssl
+, mono
 }:
 
 buildDotnetModule rec {
   pname = "jackett";
-  version = "0.19.138";
+  version = "0.20.184";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "v${version}";
-    sha256 = "0qaaccc95csahylzv65ndx990kcr075jffawbjpsjfkxzflfjq9n";
+    sha256 = "Qp1yh1bPiJyZb5myPJJqOZAbrEHl/sD+PSLxjvrj0ho=";
   };
 
   projectFile = "src/Jackett.Server/Jackett.Server.csproj";
   nugetDeps = ./deps.nix;
 
-  dotnetInstallFlags = [ "-p:TargetFramework=net5.0" ];
-  dotnet-runtime = dotnetCorePackages.aspnetcore_5_0;
+  dotnet-sdk = dotnetCorePackages.sdk_6_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_6_0;
+
+  dotnetInstallFlags = [ "-p:TargetFramework=net6.0" ];
 
   runtimeDeps = [ openssl ];
 
-  postFixup = ''
+  doCheck = !(stdenv.isDarwin && stdenv.isAarch64); # mono is not available on aarch64-darwin
+  checkInputs = [ mono ];
+  testProjectFile = "src/Jackett.Test/Jackett.Test.csproj";
+
+  postFixup = if (!stdenv.isDarwin) then ''
     # Legacy
     ln -s $out/bin/jackett $out/bin/Jackett
+  '' else ''
+    # For compatibility
+    ln -s $out/bin/Jackett $out/bin/jackett
   '';
```

ISTM that darwin build produces both binaries? https://github.com/NixOS/nixpkgs/pull/151468/checks?check_run_id=4606973723

I think we can drop the postFixup section completely (and expect user to just use `jackett` and not `Jackett`) or replace it with the following to allow both on all systems:

```suggestion
  postFixup =''
    # For compatibility
    ln -s $out/bin/jackett $out/bin/Jackett || :
    ln -s $out/bin/Jackett $out/bin/jackett || :
  '';
```

---

Sounds good, ill keep the alias for now as im not sure if anything depends on it.